### DISCUSSION TITLE
Bigger Headings in Edit Mode

### DISCUSSION THREAD

**Discussion by irmod** [damionx7/Zettel-Notes-Documentation#206] [Feature Requests]
@damionx7, please could you add a "Bigger headings" option to the Zettel Notes _Settings_ --> _Editor and Viewer_ (Edit Mode Group) to Increase text size of headers according to level (# h1, ## h2, ### h3…) in Edit Mode.

![Bigger_Headings_01.jpg](https://github.com/user-attachments/assets/64b45ea7-4c4c-45fe-919c-09fa9ec1a9f4)

I think this should be a settings item, since displaying headings can slow down the rendering of notes in edit mode.

![Bigger_Headings_02.jpg](https://github.com/user-attachments/assets/edd8dcc8-0e50-4bcd-a002-b84cdcbf8e96)

This solution improves the user experience, speeds up navigation through the text of the note, and makes the perception of the note more natural (almost WYSIWYG)

**Comment by damionx7** [comment]
I have added this feature :partying_face:

**Comment by irmod** [comment]
This feature is available from the version of Zettel Notes v2.7.8.

Settings -> Editor and Viewer -> Large Headings (Edit Mode Group)
